src/components/data_display/helpers.ts

```diff
@@ -126,11 +126,7 @@ export const buildAllColumnDefinitions = async ({
   }
 }
 
-const getFilters = (
-  outcomes: OutcomeDisplay[],
-  userId: number,
-  orgId: number | undefined
-): object => {
+const getFilters = (outcomes: DataDisplay[], userId: number, orgId: number | undefined): object => {
```

Comment:
```suggestion
const getFilters = (outcomes: DataDisplay[], userId: number, orgId: number | undefined): object => {
```

Maybe good to change the variable as well?